apps/worldcapitals/worldcapitals.star

```diff
@@ -0,0 +1,715 @@
+            display = "Americas",
+            value = "americas",
+        ),
+        schema.Option(
+            display = "Asia",
+            value = "asia",
+        ),
+        schema.Option(
+            display = "Europe",
+            value = "europe",
+        ),
+        schema.Option(
+            display = "Oceania",
+            value = "oceania",
+        ),
+    ]
+
+    return schema.Schema(
+        version = "1",
+        fields = [
+            schema.Dropdown(
+                id = "region",
+                name = "Region",
+                desc = "Limit world capitals to this region.",
+                icon = "globe",
+                default = "all",  # use all as the default
+                options = options,
+            ),
+        ],
+    )
+
+def get_cca3(country):
+    return country.get("cca3")
+
+def get_url(region):
+    if region == "all":
+        return COUNTRIES_REST_ENDPOINT_ALL
+    else:
+        return COUNTRIES_REST_ENDPOINT_REGION.format(region)
+
+####
+# FLAG DEFINITIONS
+####
+MYS_FLAG = """iVBORw0KGgoAAAANSUhEUgAAAUAAAACgCAMAAABKfUWuAAAAM1BMVEUAAGZmADPnuAkRDl/YPz/lf3/yv7/////MAAD/zAByWziSdStXRkPKoRUnH1ZAM0yuiyD/u04CAAAFLUlEQVR42u3c23LbOAwGYC6x3RYASfD9n3YvrAPlyDahOB2L+nGRaTxNZvINwANIKYQPjz8fHgGAAAQgAAEIQAACEIAABCAATwZYLImIJCs/Bvjrw+O4XdIceY1c5UcUf394HPurSFq7FTERADsiVX4YNQHwFd998sWqkswsidbMHA2Az4a+fFe1Yl/GRiUAPgrZ8un+vGFbQQLgDLNNv95MKwWAt9FvW7zdLCkB8Gv5yousa7JTmv9L5bKAG7/40kHXf1bd/fhigLpZ7b0e/XSdnXNeR1E34H8fHofGP+laLi7IMS4F7F8kDtJMKF6/ELjOasyzZeWLdmOoXb90VmHlafI1ZpvTuF4UsB0Ac3fRT0WceKKkyOmagO0AGHv3FcRTvumctHWt5YsBtu2D5+uXtp9Vp8yrk2TitoIpXQdQ+gdAyqn9sUghhHwre4rt9JMyXQcwOgqYYra1hjmHEJiZb45LBVvuHgkGAEyuFUyJyzI7M7MEYmamIOv8Q7VjJzMQYJuAPUvGpVSFmWMxZmYrK7+ww28AQPMuAS0yxzSvvrMwM0ueJ6AUXX4DALZrwL6By+ZuV2ZmjsuXPDe0Pfu58++Fmwru3UfY1G/Vbf9apg9c++HTd2PaXXD3PiJNp013J3bO3zIGoLgr+GvzehPO3dzpAat/F7xz+uRt5QwE2DsHk6gkWy/JyAu/YimJvr4OcnZAxxBY9DbfxFxVJeX9yx+iWqdLNVHL+Blo3X2EEALtq+1HTpco4eScQ+Y0fBFdyTcEYDuU9c7A+U3JNxpg7P6jn6Zhf/INAahHAPfucM2/xNvUP/tW7lAGBpKHZZzF19Y/ezPhwBj49AKm+xLm2QG9s3DfNOwYB88OaC7AJ6V7tJTPDtjuRF61oawp3Zh3E3HzcbULAIbePgBJzlVVJFkpdDuE2wGkEKgUSyKqNXek4ekBD/RTp1sI/EjwWgfrxxaC9HgZQxcDNMcg2OHnFjz/oZL/XtZ0msR615dO6u7LjgBY/TVcmZlrCfdnIqFU71g6AKC4a7gyc7YpeePtXDhOWxnLTsHzH2sSO+fhyhxl2cWYNV9DCEGiS3CAS+bqaUrfrgPS8oN6W4qX6ZsQQiB1jKYjAPpSULiWdQkZab6dRXEdQ0vtP5wb4TEHdZzqynK7LdhUtJOctWOoRbkQIPXvJJJsa7m9obqtXEnXAdyk4PMips0m8KY9w1HcrIPoQoCbjW1v7S1PN8j8M9Y1CY0IuH3OpnNDp3PBrs+JqPtixzDPyh140GG5Bl2WxKOYrwq4GQa7LpiWNVPX0xTz1/AwT2tmp6CsM27ztKa6a3icF+9U3+lu80RsXWducj8sN9Crn6rnfQnUTDXaLP+Mrgu4feb6RRLSg9V1uDLg3aXJ3Nugtu+8imcowGCx65YVbdOsEAAXGr2/YmBfb1nqO1/+NBjglyRkjlXSlGNUTHN87+vHhgMMYff6RszTnQN98xvwBgQMwfTRpSF5+wsEhwTcv0UU1cL7Y1DAEAJZ0pqn8q2afug1quMC/qUY6vV3AAQgAAEIQAACEIAABCAAAfgewH8+PD7+WPMP4nvNBBAAEIAABCACgAAEIAARBwD/RXwrPn6z/vHNBBAAEIAABCACgAAEIAARAPz7gNjNfnMvjH4K2lkABCAAEQAEIAABiAAgjjVxrIluDAKAAAQgABEABCAAAYgAII41cayJdhYCgAAEIAARAAQgAAGIuAP8hfjei3fQDkA3BoAABCACgAAEIAARAATg2eJ/ZTdjtUdp2jYAAAAASUVORK5CYII="""
```

Comment:
Not a blocker, but if you pre-scale these graphics down to their final size we'll get a little bit less data in the file. Might make it easier to work with. Looks like EXIF data is already stripped (at least from the one I checked), but that's otherwise another simple change to keep size down.